src/Hubbup.Web/Client/Standup/components.tsx

```diff
@@ -0,0 +1,370 @@
+import * as React from 'react';
+
+import * as Data from '../data';
+
+const fetchSettings: RequestInit = {
+    credentials: "same-origin",
+    headers: {
+        ["X-Requested-With"]: "XMLHttpRequest"
```

Comment:
Really minor point, but the use of `"` is inconsistent with previous lines 1 and 3. Most React devs tend to use single quotes everywhere except (for some reason I've never understood) around HTML attribute values where they tend to use double-quotes.